acceptance_tests/steps/valid_print_files.py

```diff
@@ -61,30 +62,33 @@ def rabbit_connection_and_channel():
     connection.close()
 
 
-@when('the print files are generated')
-def generate_print_files(context):
+@when('the print files are generated for "{supplier}"')
+def generate_print_files(context, supplier):
     context.print_file_paths = generate_print_files_from_config_file_path(
-        context.batch_config_path, Path('.'), context.batch_id)
+        context.batch_config_path, Path('.'), context.batch_id, supplier)
 
 
-@then('the "{manifest_count}" print files contents are valid')
-def validate_print_file_data(context, manifest_count):
+@then('the "{manifest_count}" print files contents are valid for the "{supplier}"')
+def validate_print_file_data(context, manifest_count, supplier):
     manifests = [file_path for file_path in context.print_file_paths if file_path.suffix == '.manifest']
-    print_files = [file_path for file_path in context.print_file_paths if file_path.suffix == '.csv.gpg']
+    print_files = [file_path for file_path in context.print_file_paths if file_path.suffix == '.gpg']
```

Comment:
Why this change? I don't see why it shouldn't check the `.csv` part of the suffix, it's part of the expected name format.

Reply:
I think this bit has always been broke. If you run the test and put a breakpoint there, the list comes back as empty but still passes. I'll look at checking both parts of it though because it really should

Reply:
Perhaps the `.suffix` field doesn't pick it up, `name.endswith` might be more suitable?

Reply:
Could possibly change it to this 
```python
    print_files = [file_path for file_path in context.print_file_paths if '.csv' and '.gpg' in file_path.suffixes]

```

Reply:
Worth asserting the length is expected too, since it was slipping through empty before

Reply:
Actually that logic needs a bit of a tweek, it's doing 
```python
if ('.csv') and ('.gpg' in suffixes)
```
rather than checking both strings are in `suffixes`